### ISSUE TITLE
[CS:S] Starting list changes still do not work

### ISSUE THREAD

**Issue by killer2gudpl** [lemonpole/liga-public#85]
## Describe the bug

When swapping a player in the roster (i.e. removing one and replacing with other one), the other player still does not take place. Not sure if occurs on other CS games.

## To Reproduce

1. Remove a player from starting list
2. Add another player to a starting list
3. Start a match
4. See the old roster still being there.

## Expected behavior

A new player replaces the old one.

## Screenshots

Cannot supply at the minute, will be only able in a week.

## Technical Information

- OS: Windows 10 22H2
- App Version: v3.0.0 beta-8

## Additional context

Thanks for all your work!

**Comment by lemonpole** [comment]
Hmmm I thought I fixed this. I'll look into it again.

**Comment by lemonpole** [comment]
So just making sure, the app will automatically backfill the roster when adding the bots in-game.

For example, if you have set only 3 starters it will add 1 bot so that it stays a 5v5 (4 bots + you vs 5 bots).

**Comment by killer2gudpl** [comment]
> So just making sure, the app will automatically backfill the roster when adding the bots in-game.
> 
> 
> 
> For example, if you have set only 3 starters it will add 1 bot so that it stays a 5v5 (4 bots + you vs 5 bots).

So... It's the same issue as I did send on the reddit message. Tried removing a different player but with the same outcome.

Attaching 2 older pics before this update, don't have any new ones as I am going currently away for a week.

![image](https://github.com/user-attachments/assets/0f57ff2a-9d12-457b-b2cc-baa83102d154)
![image](https://github.com/user-attachments/assets/3d698eb0-bb94-4ce4-bf63-a4acbedc2a94)
